shared/profile/dumb.js

```diff
@@ -166,7 +166,7 @@ const propsBase: RenderProps = {
   onRevokeProof: proof => console.log('onRevokeProof', proof),
   parentProps: isMobile ? {} : {
     style: {
-      width: 640,
+      width: 720,
```

Comment:
Yeah I just thought it was worth setting it to our new default width (800 window size - 80 new nav bar width).